data/persistence/src/commonMain/sqldelight/social/androiddev/common/persistence/authentication/Application.sq

```diff
@@ -1,14 +1,21 @@
 CREATE TABLE Application (
   instance Text NOT NULL PRIMARY KEY,
   client_id Text NOT NULL,
-  client_secret TEXT NOT NULL
+  client_secret TEXT NOT NULL,
+  redirect_uri TEXT NOT NULL
 );
 
 insertApplication:
-INSERT OR REPLACE INTO Application(instance, client_id, client_secret)VALUES(?,?,?);
+INSERT OR REPLACE INTO Application(instance, client_id, client_secret, redirect_uri)
+VALUES(?,?,?,?);
```

Comment:
nit: you could use named arguments here
https://cashapp.github.io/sqldelight/multiplatform_sqlite/query_arguments/#named-arguments

Reply:
Agreed, will fix in follow-up PR